### DISCUSSION TITLE
mqtt event data different than the documentation

### DISCUSSION THREAD

**Discussion by diegobaccino** [blakeblackshear/frigate#2043] [Ask A Question]
Hi all

I'm running Frigate 0.9.1 on a container on Raspberry Pi 4 32b. (v7) with a single USB TPU. I've configured my mqtt events to go to a mosquitto broker running on the same Pi as a container and then pushing the event to a InfluxDB using Telegraf. 

When I query the InfluxDB API, the schema and data for the events is different than what's in the [documentation](https://docs.frigate.video/integrations/mqtt#frigateevents). What am I missing? 

Thank you in advance. 🙏

`{
  "results": [
    {
      "statement_id": 0,
      "series": [
        {
          "name": "mqtt_consumer",
          "columns": [
            "time",
            "after_area",
            "after_box_0",
            "after_box_1",
            "after_box_2",
            "after_box_3",
            "after_end_time",
            "after_frame_time",
            "after_region_0",
            "after_region_1",
            "after_region_2",
            "after_region_3",
            "after_score",
            "after_snapshot_time",
            "after_start_time",
            "after_top_score",
            "before_area",
            "before_box_0",
            "before_box_1",
            "before_box_2",
            "before_box_3",
            "before_frame_time",
            "before_region_0",
            "before_region_1",
            "before_region_2",
            "before_region_3",
            "before_score",
            "before_snapshot_time",
            "before_start_time",
            "before_top_score",
            "event",
            "host"
          ],
          "values": [
            [
              "2021-10-17T22:19:36.427359732Z",
              22244,
              0,
              300,
              67,
              632,
              1634509176.226894,
              1634509171.011158,
              1,
              284,
              373,
              656,
              0.51171875,
              1634509157.8403,
              1634509151.601832,
              0.8046875,
              86241,
              882,
              452,
              1205,
              719,
              1634509158.839854,
              704,
              144,
              1280,
              720,
              0.65234375,
              1634509157.8403,
              1634509151.601832,
              0.779296875,
              "frigate/events",
              "cavalgo001"
            ]
          ]
        }
      ]
    }
  ]
}`

![Screenshot 2021-10-17 153038](https://user-images.githubusercontent.com/34780852/137647289-3ca31a1c-e26c-40aa-96b4-b51a5c75ad6e.png)

_______________________________________________________________
**Config File**
mqtt:
  host: localhost
cameras:
  garage:
    ffmpeg:
      #hwaccel_args:
      #  - -c:v
      #  - h264_mmal
      inputs:
        - path: rtsp://xxxxxxx
          roles:
            - detect
            - rtmp
detect:
  width: 1280
  height: 720
  fps: 5

objects:
  track:
    - person
    - horse

record:
  enabled: True
  retain_days: 0
  events:
    retain:
      default: 10

detectors:
  coral:
    type: edgetpu
    device: usb

**Comment by blakeblackshear** [comment]
Influx must be flattening the schema

**Reply by diegobaccino** [reply]
OK Thanks. Let me take a look to see if I can intercept the mqtt message in Mosquitto and see how it's coming through there. Thanks. 👍
